lib/drives/models/drive.dart

```diff
@@ -150,6 +150,7 @@ class Drive extends Trip {
   }
 
   int getUnreadMessagesCount() {
+    if (chats == null) return 0;
```

Comment:
Do we really want this? It would probably be more in line with our normal style to just require that as a precondition

Reply:
I'll remove it